test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,45 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    logStreamName=identifyLogStream['logStreams'][0]['logStreamName']
+    #print(logStreamName)
+
+    log_events_response = client.get_log_events(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,
+        logStreamName=logStreamName,
+        limit=50,
+        startFromHead=True
+    )
+
+    #print(log_events_response['events'])
+    recordsAvailable=bool(log_events_response['events'])
```

Comment:
# 152 Check log_events_response['events'] and raise exception if no logs were found. 
# 156 I don't think the print statement is necessary since you will be raising an exception. The exception message should say what happened.
# 153 & # 168 Maybe this print statement was necessary to debug the code but is it really needed?

Reply:
So for Line 153, it does seem the print is redundant, its merely a marker to see this statement appear in the logs. I realized the test case logging is not eloquent and you infact you don't see any of these print statements in the codebuild job log. Only when you go to the report and then click on each passed test case, is when you would see these. So if something has failed, it would be nice to see the line 153 printing so we know code until there is executed with no issues. So helpful when debugging. However I realize line 156 print statement is unnecessary. I will remove it.